project/accounts/templates/accounts/account_base.html

```diff
@@ -8,13 +8,13 @@
                 <div class="row top-bar sub-nav">
                     <div class="account-tabs">
                         <ul class="tabs transparent left">
-                            <li class="tab"><a class="active" href="#civis">MY CIVIS</a></li>
-                            <li class="tab"><a href="#followers">FOLLOWERS</a></li>
-                            <li class="tab"><a href="#following">FOLLOWING</a></li>
-                            <li class="tab"><a href="#issues">MY ISSUES</a></li>
+                            <li class="tab"><a class="active" href="#civis">{% trans "MY CIVIS" %} </a></li>
+                            <li class="tab"><a href="#followers">{% trans "FOLLOWERS" %}</a></li>
+                            <li class="tab"><a href="#following">{% trans "FOLLOWING" %}</a></li>
+                            <li class="tab"><a href="#issues">{% trans "MY ISSUES" %} </a></li>
```

Comment:
Let's take the opportunity to remove the all-caps style, which can be achieved via CSS if desired

```suggestion
                            <li class="tab"><a class="active" href="#civis">{% trans "My Civis" %} </a></li>
                            <li class="tab"><a href="#followers">{% trans "Followers" %}</a></li>
                            <li class="tab"><a href="#following">{% trans "Following" %}</a></li>
                            <li class="tab"><a href="#issues">{% trans "My Issues" %} </a></li>
```